plugin/helpers.go

```diff
@@ -18,6 +18,36 @@ import (
 // DefaultTimeout is the default timeout for prefetching
 var DefaultTimeout = 10 * time.Second
 
+/*
+CName returns the entry's canonical name, which is what Wash uses
+to construct the entry's path (see plugin.Path). The entry's cname
+is e.Name(), but with all '/' characters replaced by a '#' character.
+CNames are necessary because it is possible for entry names to have '/'es
+in them, which is illegal in bourne shells and UNIX-y filesystems.
+
+CNames are unique. CName uniqueness is checked in plugin.CachedList.
+
+NOTE: The '#' character was chosen because it is unlikely to appear in
+a meaningful entry's name. If, however, there's a good chance that an
+entry's name can contain the '#' character, and that two entries can
+have the same cname (e.g. 'foo/bar', 'foo#bar'), then you can use
+e.SetSlashReplacementChar(<char>) to change the default slash replacement
+character from a '#' to <char>.
+*/
+func CName(e Entry) string {
```

Comment:
Why is this a separate function rather than directly exporting the `CName` method on `EntryBase`?

Reply:
Oh, is it because you don't want someone to be able to override it? Can you add a comment to that effect on `cname`?